### ISSUE TITLE
loading

### ISSUE THREAD

**Issue by chphoom** [chphoom/aroundtheblock#74]
#69 -> loading works on dev but when generating a new challenge a second time but loading circle appears above the previously generated challenge

**Comment by elaine331917** [comment]
is this a big issue? do we want the previously generated challenge to disappear?

**Comment by chphoom** [comment]
> is this a big issue? do we want the previously generated challenge to disappear?

not really, i think it makes sense for just the loading spinner to be there tho

u can mark this as wont fix and close the issue if u think it's unimportant
